runtime/centrifuge/src/lib.rs

```diff
@@ -137,14 +137,18 @@ impl Contains<Call> for BaseFilter {
 			c,
 			// Calls from Sudo
 			Call::Sudo(..)
-			// Calls for runtime upgrade
-			| Call::System(frame_system::Call::set_code{..})
-			| Call::System(frame_system::Call::set_code_without_checks{..})
+			| Call::System{..}
```

Comment:
Since benchmarks require System in most of the added ones, we might as well enable it since all the extrinsics in the System pallet are sudo or remark